node/bindnode/infer.go

```diff
@@ -87,6 +104,78 @@ func fieldNameFromSchema(name string) string {
 	return strings.Title(name)
 }
 
+var defaultTypeSystem schema.TypeSystem
+
+func init() {
+	defaultTypeSystem.Init()
+
+	defaultTypeSystem.Accumulate(schemaTypeBool)
+	defaultTypeSystem.Accumulate(schemaTypeInt)
+	defaultTypeSystem.Accumulate(schemaTypeFloat)
+	defaultTypeSystem.Accumulate(schemaTypeString)
+	defaultTypeSystem.Accumulate(schemaTypeBytes)
+	defaultTypeSystem.Accumulate(schemaTypeLink)
```

Comment:
I think it may be a good time to plonk all these into a single helper function over in the schema package (or possibly in the near future, `schema/compiler`, if things get factored out that way) called `schema.AccumulateStandardPrelude`.

Reply:
Separated into https://github.com/ipld/go-ipld-prime/issues/208 :)